lineman/app/components/group_page/group_page.scss

```diff
@@ -59,12 +59,23 @@
   @include card;
 }
 
+.group-page__threads {
+  padding: 10px 0 17px;
+  h2 {
+    display: inline-block;
+    margin: 0 17px 17px 17px;
+    padding-top: 7px;
+  }
+}
+
 .group-page__new-thread{
-  float: right;
-  margin-top: -5px;
-  margin-bottom: 10px;
-  .btn { @include fontSmall; }
+  .btn {
```

Comment:
I think the correct BEM name here is .group-page__new-thread-button (rather than nesting .btn)